prometheus_client/metrics.py

```diff
@@ -553,7 +553,7 @@ def __init__(self,
         )
         self._kwargs['buckets'] = buckets
 
-    def _prepare_buckets(self, buckets):
+    def _prepare_buckets(self, buckets: Sequence[float]) -> None:
```

Comment:
We specifically cast each element in buckets to a float on the next line. Should this be something like a union of int/float/str instead?

Reply:
I thought about the same :)

Reply:
I've push a fix.